testing/integration/upgrade_test.go

```diff
@@ -70,6 +70,7 @@ func (s *UpgradeElasticAgent) SetupSuite() {
 }
 
 func (s *UpgradeElasticAgent) TestUpgradeFleetManagedElasticAgent() {
+	kibClient := s.requirementsInfo.KibanaClient
```

Comment:
This change is just some refactoring, unrelated to the FQDN test.  If you prefer, I can pull this change out into its own PR.